android/res/values-es/strings.xml

```diff
@@ -579,8 +579,8 @@
 	<string name="bookmark_lists_hide_all">Ocultar todo</string>
 	<string name="bookmark_lists_show_all">Mostrar todo</string>
 	<string name="bookmarks_create_new_group">Crear nueva lista</string>
-	<!-- Bookmark categories screen -->
-	<string name="bookmarks_import">Importar marcadores</string>
+	<!-- Bookmark categories screen, button that opens folder selection dialog to import KML/KMZ/GPX/KMB files -->
+	<string name="bookmarks_import">Importar marcadores y pistas</string>
```

Comment:
Apologies for the extremely late comment, but can we change this translation of track to “trazas” (and “traces” in Catalan; singular “traça”) to be consistent with OSM?